### PR TITLE
changed log rocket creds

### PR DESCRIPTION
**Pull request by ryanprice**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ import { store } from './store/'
 import LogRocket from 'logrocket';
 
 if (process.env.NODE_ENV == "development") {
-  LogRocket.init('hkkfms/cryptoztest');
+  LogRocket.init('jf4zzm/cryptoz-testbsc');
 } else {
-  LogRocket.init('hkkfms/cryptozbsc');
+  LogRocket.init('jf4zzm/the-cryptoz-nft-universe');
```
Comment (mchigit) [inline src/main.js:15]: I'm wondering if we should use environment variable for stable logrocket cred? Can we let anyone to see this?

Comment (ryanprice) [reply src/main.js:15]: yes ! we should

Comment (jjjhill) [reply src/main.js:15]: this shouldn't really matter. its not a security threat or anything

**Review by jjjhill — APPROVED** [review]

File: src/main.js (hunk 1 above)
Comment (ryanprice) [reply src/main.js:15]: We should do a env split anyways.. there may be more things we don't in the public to come, the Google stuff for example now

Comment (jjjhill) [reply src/main.js:15]: no i agree, Michael is suggesting we remove 'jf4zzm/the-cryptoz-nft-universe' from the source code and put it in an environment variable on our systems.. but i dont think its neccessary

**Pull request merged**